apps/web/src/client/components/file-uploader/_server-actions.js

```diff
@@ -92,11 +96,13 @@ const serverActions = (uploadForm) => {
 		let response;
 
 		try {
-			const blobClient = containerClient.getBlockBlobClient(blobStoreUrl);
+			const blobClient = containerClient.getBlockBlobClient(blobStoreUrl.slice(1));
```

Comment:
After everything is working, let's update the backend so that there so need for this `slice` function here

Reply:
absolutely. i'll put a todo somewhere